src/ansys/dpf/gate/dpf_vector.py

```diff
@@ -33,13 +33,18 @@ class DPFVectorBase:
 
     def __init__(self, owner, api):
         self.dpf_vector_api = api
-        self._modified = False
-        self._check_changes = True
+
+        # The updated version of the DPF vector will always be committed to DPF.
+        # Ideally, this should be set to True only when modified, however this is not possible to do that efficiently.
+        # Consequently, for performance reasons, it's much better to always commit the vector to DPF rather than
```

Comment:
@cbellot000 is this also true in the case of a huge vector in gRPC on a remote machine with a high ping?

Reply:
since it's twice faster, I think that it should be safe to assume that it will as fast